notifications.go

```diff
@@ -26,6 +26,10 @@ func NewNotificationManager(state *AppState) (*NotificationManager, error) {
 }
 
 func (n *NotificationManager) ShouldNotify(reply *forest.Reply) bool {
+	if uint64(reply.Created) < n.AppState.TimeLaunched {
+		// do not send old notifications
+		return false
+	}
```

Comment:
This is the actual comparison which blocks old notifications. I couldn't figure out how to store `TimeLaunched` as a timestamp type so I converted the reply's timestamp to match. I realize this is a little odd but it works great.

Reply:
Thanks for putting this in @Caton101! One thing I'm not sure about is if we want to notify people of messages sent since the last time they opened the chat. That's something that we should discuss. Our primary form of development communication is through our [ticket tracker](https://todo.sr.ht/~whereswaldon/arbor-dev) and out [mailing list](https://lists.sr.ht/~whereswaldon/arbor-dev). There is a [ticket open](https://todo.sr.ht/~whereswaldon/arbor-dev/113) to discuss the issue. If you have a chance, we'd love if you could reply to the ticket for visibility; there may be some more discussion to happen there. Thanks again man :)

Reply:
I am aware of both of those links. My issue is that I do not have the ability to interact with either of them. I would love to hear from @whereswaldon on how I can interact with the ticket system.

Reply:
@aThorp96 I got some help moving to sr.ht. Please see my comment at https://todo.sr.ht/~whereswaldon/arbor-dev/113#comment-12823